cv32/sim/uvmt_cv32/xrun.mk

```diff
@@ -113,6 +113,13 @@ dhrystone: comp $(CUSTOM)/dhrystone.hex
 		+UVM_TESTNAME=uvmt_cv32_firmware_test_c \
 		+firmware=$(CUSTOM)/dhrystone.hex
 
+riscv_ebreak_test_0: comp $(CUSTOM)/riscv_ebreak_test_0.hex
```

Comment:
Same as previous comment.

Reply:
New code is the ebreak . S test.

When I try to make the riscv_ebreak_test_0 using make custom this is what I get:
make custom CUSTOM_PROG=riscv_ebreak_test_0 SIMULATOR=xrun XRUN_USE_ISS=YES CV32E40P_PKG=/wrk/gtumbush/mcu/OpenHW/cv32e40p/master

make: *** No rule to make target `/wrk/gtumbush/mcu/OpenHW/regression_test/cv32/sim/uvmt_cv32/../../../cv32/tests/core/custom/riscv_ebreak_test_0.c', needed by `/wrk/gtumbush/mcu/OpenHW/regression_test/cv32/sim/uvmt_cv32/../../../cv32/tests/core/custom/riscv_ebreak_test_0.elf'.  Stop.

.S tests don’t seem to be supported as a custom test.

Reply:
Ah!   Good catch.  I'll take an action to figure out how to support the assembler (*.S) custom test-programs.   We'll want that for riscv-dv support for sure.   In the meantime, let's bring in your `riscv_ebreak_test_0` target as a temporary fix.